setup.py

```diff
@@ -10,7 +10,7 @@
 ]
 setup(
     name="dataflux-pytorch",
-    packages=["dataflux_core", "dataflux_pytorch"],
+    packages=["dataflux_core", "dataflux_pytorch", "dataflux_pytorch.lightning"],
```

Comment:
Is this line causing the problem? Ideally we should be using `find_packages()` ([ref](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#packages)) to find the packages so that we don't need to specify this every time a new package is added (error-prone).

We probably took a short path because we needed "dataflux_core" to map to "dataflux_client_python/dataflux_core" (see below). Ideally we should be just importing "dataflux_client_python.dataflux_core" instead of "dataflux_core". Could you investigate if that path is feasible?

Reply:
I see we are reverting to not using "find_packages()", which is fine. But could you file a follow-up ticket to investigate the solutions here? Ideally we would not want every extra directory (package) to be declared every time they are added?